modules/order/templates/commerce-order--default.html.twig

```diff
@@ -0,0 +1,77 @@
+{#
+/**
+ * @file
+ * Default order template.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the wrapper.
+ * - order: The rendered order fields.
+ *   Use 'order' to print them all, or print a subset such as
+ *   'order.order_number'. Use the following code to exclude the
+ *   printing of a given field:
+ *   @code
+ *   {{ order|without('order_number') }}
+ *   @endcode
+ * - order_entity: The order entity.
+ *
+ * @ingroup themeable
+ */
+#}
+
+{{ attach_library('commerce_order/form') }}
+{% set order_state = order_entity.getState.value|capitalize %}
```

Comment:
You're capitalizing the machine name of the state, instead of taking the label. That's not right.